### ISSUE TITLE
The good and the bad: Are some attribute words better than others in the Implicit Association Test?

### ISSUE THREAD

**Issue by ben-domingue** [ben-domingue/irw#61]
https://osf.io/ezj5t/?view_only=1781c05b04d54b829fd2eff67e0d429c
paper. https://link.springer.com/article/10.3758/s13428-021-01592-8#author-information

**Comment by rn2407stanford** [comment]
This is a little bit hard dataset for me. So, 
1) Do we need to exclude demographic variables if they are in likert scale? (e.g., "To what extent do you identify as Arab-Muslim" and then asks the actual item "On average, how much do you like Arab Muslims?").
2) It has session ID (those are unique) and then it has participant ID (those are not unique as participants can participate multiple times). Do we need to keep both of them?

[Study2_Codebook.csv](https://github.com/ben-domingue/irw/files/15327544/Study2_Codebook.csv)
[Study2_Data.csv](https://github.com/ben-domingue/irw/files/15327545/Study2_Data.csv)

**Comment by ben-domingue** [comment]
i'd be inclined to just take the first observation of each uniqueid. so just duplicate_id!=1
thoughts about that? 
we can then make the uniqueid the 'id'

**Comment by rn2407stanford** [comment]
Results 
[IAT_study2.xlsx](https://github.com/ben-domingue/irw/files/15343646/IAT_study2.xlsx)
[IAT_study3.xlsx](https://github.com/ben-domingue/irw/files/15343647/IAT_study3.xlsx)

**Comment by rn2407stanford** [comment]
Yes. If you look at this supplement material page 22 is for the study-2 and page 25 is for study-3. 
(In the article also it says study-2 compares only best and worst performing words, and study-3 tests variability and relevance of attribute words to the attribute categories)

[OnlineSupplement (1).pdf](https://github.com/ben-domingue/irw/files/15404545/OnlineSupplement.1.pdf)
